havoc.c

```diff
@@ -135,16 +135,16 @@ int test_havoc(){
           return ++errors;
         }
 
+        if (verbose)
+            Printf("Process %d changing allocation : %d -> %d, current allocation: %d%% \n", pid, processes[pid].request, new_allocation, (100-remaining_allocation - processes[pid].request + new_allocation));
```

Comment:
(100-remaining_allocation - processes[pid].request + new_allocation) computes the current allocation after making the request. 
Since the new_remaining_allocation will be (remaining_allocation + processes[pid].request - new_allocation),
the current allocation of CPU = 100 - new_remaining_allocation = 100 - (remaining_allocation + processes[pid].request - new_allocation)